lightning/src/ln/channel.rs

```diff
@@ -850,8 +854,11 @@ impl<Signer: Sign> Channel<Signer> {
 		let holder_signer = keys_provider.get_channel_signer(false, channel_value_satoshis);
 		let pubkeys = holder_signer.pubkeys().clone();
 
-		if channel_value_satoshis >= MAX_FUNDING_SATOSHIS {
-			return Err(APIError::APIMisuseError{err: format!("funding_value must be smaller than {}, it was {}", MAX_FUNDING_SATOSHIS, channel_value_satoshis)});
+		if !their_features.supports_wumbo() && channel_value_satoshis >= MAX_FUNDING_SATOSHIS_NO_WUMBO {
```

Comment:
Lets not bother with the feature check - it doesn't change anything to add it, but adds a good chunk of additional code and constant(s) to test for.